schema.graphql

```diff
@@ -1120,14 +1120,16 @@ type StakeExchangeOfferCreated {
   rate: BigInt!
   minAverageCommission: Int! @deprecated
   minAverageRewardRate: BigInt!
-  tokenFilter: StakeExchangeTokenFilter 
+  tokenFilter: StakeExchangeTokenFilter
+  state: PoolState!
```

Comment:
are you going to use the state here? it seems not added to the event for this action